src/Components/Ignitor/src/Ignitor.csproj

```diff
@@ -1,10 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <!-- Intentionally targeting netcoreapp3.0 because we do not want to update the Benchmark client to use 3.1 -->
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
This part wasn't resolved as far as I know:
> @sebastienros will the TFM change break the benchmark client❔

Reply:
Benchmarks uses a fixed version of the client so it should be fine for now: https://github.com/aspnet/Benchmarks/blob/master/src/BenchmarksClient/BenchmarksClient.csproj#L30. We were planning on updating the Blazor benchmarks to use crank so this would be an added incentive.

Reply:
Forgot we are all @sebastienros 😃 Thanks @pranavkm❕

Reply:
Thanks